src/common/constants.d

```diff
@@ -102,83 +102,97 @@ immutable string[] scopes = [
  * Compiler-defined values for version() conditions.
  */
 immutable string[] predefinedVersions = [
-	"AArch64",
-	"AIX",
-	"all",
-	"Alpha",
-	"Alpha_HardFloat",
-	"Alpha_SoftFloat",
-	"Android",
-	"ARM",
-	"ARM_HardFloat",
-	"ARM_SoftFloat",
-	"ARM_SoftFP",
-	"ARM_Thumb",
-	"assert",
-	"BigEndian",
-	"BSD",
-	"Cygwin",
-	"D_Coverage",
-	"D_Ddoc",
-	"D_HardFloat",
-	"DigitalMars",
-	"D_InlineAsm_X86",
-	"D_InlineAsm_X86_64",
-	"D_LP64",
-	"D_NoBoundsChecks",
-	"D_PIC",
-	"DragonFlyBSD",
-	"D_SIMD",
-	"D_SoftFloat",
-	"D_Version2",
-	"D_X32",
-	"FreeBSD",
-	"FreeStanding",
-	"GNU",
-	"Haiku",
-	"HPPA",
-	"HPPA64",
-	"Hurd",
-	"IA64",
-	"LDC",
-	"linux",
-	"LittleEndian",
-	"MIPS32",
-	"MIPS64",
-	"MIPS_EABI",
-	"MIPS_HardFloat",
-	"MIPS_N32",
-	"MIPS_N64",
-	"MIPS_O32",
-	"MIPS_O64",
-	"MIPS_SoftFloat",
-	"NetBSD",
-	"none",
-	"OpenBSD",
-	"OSX",
-	"Posix",
-	"PPC",
-	"PPC64",
-	"PPC_HardFloat",
-	"PPC_SoftFloat",
-	"S390",
-	"S390X",
-	"SDC",
-	"SH",
-	"SH64",
-	"SkyOS",
-	"Solaris",
-	"SPARC",
-	"SPARC64",
-	"SPARC_HardFloat",
-	"SPARC_SoftFloat",
-	"SPARC_V8Plus",
-	"SysV3",
-	"SysV4",
-	"unittest",
-	"Win32",
-	"Win64",
-	"Windows",
-	"X86",
-	"X86_64",
+    "AArch64",
+    "AIX",
+    "all",
+    "Alpha",
+    "Alpha_HardFloat",
+    "Alpha_SoftFloat",
+    "Android",
+    "ARM",
+    "ARM_HardFloat",
+    "ARM_SoftFloat",
+    "ARM_SoftFP",
+    "ARM_Thumb",
+    "assert",
+    "BigEndian",
+    "BSD",
+    "CRuntime_Bionic",
+    "CRuntime_DigitalMars",
+    "CRuntime_Glibc",
+    "CRuntime_Microsoft",
+    "Cygwin",
+    "DigitalMars",
+    "DragonFlyBSD",
+    "D_Coverage",
+    "D_Ddoc",
+    "D_HardFloat",
+    "D_InlineAsm_X86",
+    "D_InlineAsm_X86_64",
+    "D_LP64",
+    "D_NoBoundsChecks",
+    "D_PIC",
+    "D_SIMD",
+    "D_SoftFloat",
+    "D_Version2",
+    "D_X32",
+    "ELFv1",
+    "ELFv2",
+    "Epiphany",
+    "FreeBSD",
+    "FreeStanding",
+    "GNU",
+    "Haiku",
+    "HPPA",
+    "HPPA64",
+    "Hurd",
+    "IA64",
+    "iOS",
+    "LDC",
+    "linux",
+    "LittleEndian",
+    "MinGW",
+    "MIPS32",
+    "MIPS64",
+    "MIPS_EABI",
+    "MIPS_HardFloat",
+    "MIPS_N32",
+    "MIPS_N64",
+    "MIPS_O32",
+    "MIPS_O64",
+    "MIPS_SoftFloat",
+    "NetBSD",
+    "none"
```

Comment:
That's embarassing. What has happened is that
- I've copied from dmd
- I've pasted in an editor, I sorted the selection and removed the last comma
- I've see that in DCD the list is sorted but not case-sensitively
- I've sorted again with other settings

=> "none" was not at the end of list anymore